### PR TITLE
Partial rendering of INDYWIDUALNE decision

### PR DESCRIPTION
**Pull request by magul**
How to test:
* Open `src/docx/index.js` and change values in `generateDocument` execution second parameter.
* `npm install`
* `node src/docs/index.js`
* Open `output.docx`.

There are still some options to be renered based on `data but that should be proper start to refine on.

### CODE REVIEW

File: src/docx/assets/indywidualne/docProps/core.xml [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
-  <dcterms:created xsi:type="dcterms:W3CDTF">2019-07-14T15:37:29Z</dcterms:created>
-  <dc:creator/>
-  <dc:description/>
-  <dc:language>en-US</dc:language>
-  <cp:lastModifiedBy/>
-  <cp:revision>0</cp:revision>
-  <dc:subject/>
   <dc:title/>
+  <dc:subject/>
+  <dc:creator>python-docx</dc:creator>
```
Comment (Rafalkufel) [inline src/docx/assets/indywidualne/docProps/core.xml]: We should remember to remove that `python` description in the future.

File: src/docx/assets/indywidualne/word/fontTable.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+<?xml version='1.0' encoding='UTF-8' standalone='yes'?>
+<w:fonts mc:Ignorable="w14" xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships" xmlns:w="http://schemas.openxmlformats.org/wordprocessingml/2006/main" xmlns:w14="http://schemas.microsoft.com/office/word/2010/wordml">
+  <w:font w:name="Symbol">
+    <w:panose1 w:val="00000000000000000000"/>
+    <w:charset w:val="02"/>
+    <w:family w:val="auto"/>
+    <w:pitch w:val="variable"/>
+    <w:sig w:csb0="80000000" w:csb1="00000000" w:usb0="00000000" w:usb1="10000000" w:usb2="00000000" w:usb3="00000000"/>
```
Comment (Rafalkufel) [inline src/docx/assets/indywidualne/word/fontTable.xml]: I think we can remove all fonts except `TimesNewRoman` and `Symbols` (maybe)

File: src/docx/assets/indywidualne/docProps/core.xml (hunk 1 above)
Comment (magul) [reply src/docx/assets/indywidualne/docProps/core.xml]: We will. It also does not properly render all fields (as I don't fully understand the logic behind it). That's just first iteration here. Will refine on it over time (creating also a lot of small tickets to be done in no-time).

File: src/docx/assets/indywidualne/word/fontTable.xml (hunk 2 above)
Comment (magul) [reply src/docx/assets/indywidualne/word/fontTable.xml]: Agree, but not in that pull request. I can create separate issues for:
 * `docProps/core.xml` to contain `alinka.io` as a generator and proper time of generation
 * refine set of fonts that we, in fact, going to use in our rendered documents.

This pull request is only about proving that using templates should be an efficient way to generate documents based on provided values.

**Pull request merged**